plugins/node/opentelemetry-instrumentation-ioredis/src/utils.ts

```diff
@@ -121,6 +121,12 @@ export const traceSendCommand = (
       const origResolve = cmd.resolve;
       /* eslint-disable @typescript-eslint/no-explicit-any */
       cmd.resolve = function (result: any) {
+        try {
```

Comment:
could you use https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-instrumentation/src/utils.ts#L24 so we are sure we can't crash user code ?

Reply:
Thanks, food point. Updated the PR